source/includes/app.md

```diff
@@ -115,5 +115,186 @@ app.sai.teleport();
 ```javascript
 app.sai.animate('Happy');
 ```
-
 The `sai` object can be used to control the Spatial Artificial Intelligence familiar. Use SAI to communicate essential information about a scene or direct a player's attention toward an important element.
+
+
+> Changes visibility of SAI
+
+```javascript
+app.sai.schema.setBool('visible', false);
+```
+> Changes font size of SAI text
+
+```javascript
+app.sai.schema.setNumber('fontSize', 70f);
+```
+> Changes the magnet tolerance of sai when it is unfocused. SAI will readjust its position only when the gaze spot is beyond the specified threshold.  
+
+```javascript
+app.sai.schema.setNumber('magnetTolerance', 2f);
```

Comment:
Ditto here, just `2` is fine, or something like `2.2` if the underlying value is a float.